packages/ts-scripts/src/webpack/webpack.config.dev.js

```diff
@@ -15,160 +15,161 @@ const generateRegexExcludePackages = () => {
   return new RegExp(`node_modules/(?!(${listPackagesString})/).*`)
 }
 
-const tagName = getVersionTag()
-
-const webpackConfigDev = ({ appName }) => ({
-  mode: 'development',
-  target: 'web',
-  bail: true,
-  devtool: 'inline-source-map',
-  context: process.cwd(),
-  entry: [PATHS.entryWeb],
-  output: {
-    pathinfo: false,
-    path: PATHS.output,
-    filename: '[name].[contentHash].js',
-  },
-  plugins: [
-    new NodePolyfillPlugin(),
-    new ForkTsCheckerWebpackPlugin({
-      eslint: {
-        files: './src/**/*.{ts,tsx,js,jsx}',
-      },
-    }),
-    new ResolveTSPathsToWebpackAlias({
-      tsconfig: PATHS.tsConfig,
-    }),
-    new EnvironmentPlugin({
-      APP_VERSION: `${tagName.packageName}_${tagName.version}`,
-    }),
-    new HtmlWebpackPlugin({
-      inject: true,
-      template: PATHS.template,
-    }),
-    new FaviconsWebpackPlugin({
-      logo: PATHS.logo,
-      cache: true,
-      inject: true,
-      mode: 'light',
-      devMode: 'light',
-      favicons: {
-        appName,
-        developerName: 'Reapit Ltd',
-        theme_color: '#262F69',
-        icons: {
-          favicons: true,
-          android: false,
-          appleIcon: false,
-          appleStartup: false,
-          coast: false,
-          firefox: false,
-          opengraph: false,
-          twitter: false,
-          yandex: false,
-          windows: false,
+const webpackConfigDev = ({ appName }) => {
```

Comment:
Feels a bit scary because this is used by all the things - bit hard to follow what changed - can you just fill me in in the morning please? Ta!